net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
```

Comment:
Well, I am not a lawyer but these days it is recommended that people have their own name in the copyright header unless there is a legal contract between them and the named entity requiring otherwise.